oddt/scoring/__init__.py

```diff
@@ -63,6 +80,81 @@ def __init__(self, model_instance, descriptor_generator_instance, score_title='s
         self.descriptor_generator = descriptor_generator_instance
         self.score_title = score_title
 
+    def _gen_pdbbind_desc(self,
```

Comment:
Right now we train all of our scoring functions on PDBbind, but I think we should separate the main `scorer` class from `pdbbind_training` scorer class (name subjected to change).